### ISSUE TITLE
Request: add lf

### ISSUE THREAD

**Issue by MaxVerevkin** [chaotic-aur/packages#184]
`lf` - [AUR](https://aur.archlinux.org/packages/lf/)

**Comment by nemya9066** [comment]
Yes please show some love for the lf file manager it's even more popular than ranger nowadays.
Thanks in advance :)

**Comment by PedroHLC** [comment]
I've added this package, but it'll have to keep this issue open until I push it to this repo...
